Amendments 2023 (#59)
uio-carpentry/uio-carpentry.github.io: content/about/charter.md

### Secretary

- Ensure that someone is assigned to and takes minutes during all
    board meetings

> - Report decisions and action items in formal minutes published

Reviewer: Since it doesn't need to be the secretary who takes minutes (first point), how about: 
```suggestion
- Ensure that decisions and action items are reported in formal minutes published publicly
```

>     publicly

Reviewer: ```suggestion

```

- Revise minutes for publication, and publish minutes

### Communications officer

- Receive all communication from community to the board and relay as

>   necessary

Author: ```suggestion
  necessary
  - Mentoring instructor trainees towards certification and teaching
```

### Community events officer

> - Organise regular community events

Author: ```suggestion
- Organise regular community events, for example once per semester
```

### Elections officer

- Recruit community members to stand for election
- Coordinate the election
- This role should preferably be held by a board member 
    rotating off the board at the next election

## Elections

The criteria for individuals to stand for election to the board are:

- Is a member of the Carpentry@UiO, broadly defined
- Has participated in a Carpentries event in any capacity (learner,
    helper, instructor, organizer, board member or host) at UiO or elsewhere in the
    past 2 years

Board members will not serve for more than two consecutive three year terms,
followed by at least a one year hiatus before serving any new term(s).

Anyone who has participated in a Carpentries event in any capacity

> (learner, helper, instructor, organizer, or host) at UiO or elsewhere in

Reviewer: This seems somewhat more rigidly defined than the criteria to stand for election to the board above. Intentional? Add "board member" as above?

> (learner, helper, instructor, organizer, or host) at UiO or elsewhere in

Author: ```suggestion
(learner, helper, instructor, organizer, board member or host) at UiO or elsewhere in
```

the past 2 years will be eligible to vote.

Elections will be held at an annual community meeting each year

> taking place in November or December.

Reviewer: Maybe some flexibility here?
```suggestion
preferably taking place in November or December.
```
